lib/board_selection.py

```diff
@@ -0,0 +1,15 @@
+from .tictactoe_board import TicTacToeBoard
+
+
+class BoardSelection:
+    def __init__(self, ui):
+        self.ui = ui
+
+    def select_board_size(self):
+        self.ui.display("Welcome to TIC TAC TOE!")
+        self.ui.display("First, select a board size.")
+        size = int(self.ui.get_user_input())
+        if size != 3 and size != 4:
```

Comment:
Instead of hardcoding these magic numbers, could we store them in a list inside this class and do something like:

```py
if size not in valid_board_sizes:
```

Reply:
To follow up: it's much easier to add a valid board size to the `valid_board_sizes` list rather than adding another check in this `if` statement. For example, if there was a requirement to add 5x5, 6x6, 7x7, and 8x8 board sizes, then this `if` statement is going to get bigger and bigger.

Reply:
Remember, we want to add new code without breaking existing functionality 😉

Reply:
Done!